workflow: sign release containers (#1806)
intel/intel-device-plugins-for-kubernetes: README.md

[imagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/#updating-images)
```IfNotPresent``` and can be changed with ```scripts/set-image-pull-policy.sh```.

### Signed container images

> Starting from 0.31 release, the images (`0.31.0` etc., not `devel`) are signed with keyless signing using `cosing`. The signing proof is stored in [rekor.sigstore.dev](https://rekor.sigstore.dev) in an append-only transparency log. The signature is also stored within the dockerhub.

Reviewer: ```suggestion
Starting from 0.31 release, the images (`0.31.0` etc., not `devel`) are signed with keyless signing using `cosign`. The signing proof is stored in [rekor.sigstore.dev](https://rekor.sigstore.dev) in an append-only transparency log. The signature is also stored within the dockerhub.
```

Author: ah, yes, let's all co-sing some songs.

> Beginning of 0.31 release, release images (`0.31.0` etc., not `devel`) will be signed with keyless signing using `cosing`. The signing proof is stored in [rekor.sigstore.dev](https://rekor.sigstore.dev) in an append-only transparency log. The signature is also stored within the dockerhub.

Reviewer 2: small edits perhaps:
```suggestion
Starting from 0.31 release, the images (`0.31.0` etc., not `devel`) are signed with keyless signing using `cosing`. The signing proof is stored in [rekor.sigstore.dev](https://rekor.sigstore.dev) in an append-only transparency log. The signature is also stored within the dockerhub.
```
